### PR TITLE
Update library urls for ESPEssentials and FastLEDHub

### PR DESCRIPTION
**Pull request by srwi**
After a GitHub username change the library urls for ESPEssentials and FastLEDHub have been updated.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @srwi. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/04/18 15:01:37 Loaded 6842 libraries from DB
Changing URL of library ESPEssentials from https://github.com/stnkl/ESPEssentials.git to https://github.com/srwi/ESPEssentials.git
Success!
```
```
2024/04/18 14:59:57 Loaded 6842 libraries from DB
Changing URL of library FastLEDHub from https://github.com/stnkl/FastLEDHub.git to https://github.com/srwi/FastLEDHub.git
Success!
```

**Pull request merged**
